client/src/slate-helpers/slate-change-mutations/insertPointerExport.ts

```diff
@@ -37,6 +38,14 @@ export function insertPointerExport(change: Change) {
     slateChangeMutations.moveSelectionAwayFromPointerEdge(change);
   }
 
+  // disallow attempts to export across pointer boundaries
+  const selection = change.value.selection;
+  const document = change.value.document;
+  const isExportingAcrossPointers = isSelectionAcrossPointers(selection, document);
```

Comment:
what about passing ``value`` in instead, then not having to make those two const declarations above? Are you expecting to use ``isSelectionAcrossPointers`` in areas where value is not available?